### DISCUSSION TITLE
Need Help !!!!!!!!!!!!!!! Copied component navbar responsive from daisy stories but not work as expected?

### DISCUSSION THREAD

**Discussion by hduynam99** [saadeghi/daisyui#1208] [Q&A] [answered]
Why I am not getting the UI looks like on daisy UI stories. here is my screen shot on my screen
![image](https://user-images.githubusercontent.com/56583073/194115604-a43634ac-fe44-4efe-9724-6b5124381555.png)

the code is exactly the same with the stories.

Here is what is suppose to look like: 
https://react.daisyui.com/?path=/story/navigation-navbar--responsive&args=dataTheme:luxury

Thanks!!

**Comment by saadeghi** [comment, marked as answer]
Please open an issue here https://github.com/daisyui/react-daisyui

react-daisyui is an experimental community project and I didn't work on it myself.

**Comment by MatthewLaFalce** [comment]
@saadeghi I have seen the same behavior using the regular HTML code snipped copied from the [documentation site](https://daisyui.com/components/navbar/#responsive-dropdown-menu-on-small-screen-center-menu-on-large-screen)


### Actual
<img width="686" alt="Screen Shot 2022-10-27 at 10 53 36 PM" src="https://user-images.githubusercontent.com/42977644/198483290-b499ca73-ac2c-4a5c-a929-37d13824d89b.png">


### Expected
![Screen Shot 2022-10-27 at 10 56 19 PM](https://user-images.githubusercontent.com/42977644/198483985-020c8960-4cf5-4fbd-bcde-9f4b70982621.png)


### My Code

```html
<div class="navbar bg-base-100">
  <div class="navbar-start">
    <div class="dropdown">
      <label tabindex="0" class="btn btn-ghost lg:hidden">
        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" fill="none" viewBox="0 0 24 24" stroke="currentColor"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h8m-8 6h16" /></svg>
      </label>
      <ul tabindex="0" class="menu menu-compact dropdown-content mt-3 p-2 shadow bg-base-100 rounded-box w-52">
        <li><a>Item 1</a></li>
        <li tabindex="0">
          <a class="justify-between">
            Parent
            <svg class="fill-current" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M8.59,16.58L13.17,12L8.59,7.41L10,6L16,12L10,18L8.59,16.58Z"/></svg>
          </a>
          <ul class="p-2">
            <li><a>Submenu 1</a></li>
            <li><a>Submenu 2</a></li>
          </ul>
        </li>
        <li><a>Item 3</a></li>
      </ul>
    </div>
    <a class="btn btn-ghost normal-case text-xl">daisyUI</a>
  </div>
  <div class="navbar-center hidden lg:flex">
    <ul class="menu menu-horizontal p-0">
      <li><a>Item 1</a></li>
      <li tabindex="0">
        <a>
          Parent
          <svg class="fill-current" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"><path d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"/></svg>
        </a>
        <ul class="p-2">
          <li><a>Submenu 1</a></li>
          <li><a>Submenu 2</a></li>
        </ul>
      </li>
      <li><a>Item 3</a></li>
    </ul>
  </div>
  <div class="navbar-end">
    <a class="btn">Get started</a>
  </div>
</div>
```

**Reply by saadeghi** [reply]
This is not the same issue.
I can't be sure what is the cause of problem unless you share your project but make sure:
1. You installed daisyUI `npm i daisyui`
2. You have daisyUI as a plugin in `tailwind.config.js` file: `plugins: [require("daisyui")],`
3. In your `tailwind.config.js` file you have the path to all your markup files in `content` object.
  Something like this:
    ```js
      module.exports = {
        content: [
          './pages/**/*.{html,js}',
          './components/**/*.{html,js}',
        ],
      }
    ```
    The path to every file that contains a Tailwind CSS class name or a daisyUI class name must be included in `content`.
    Read more: https://tailwindcss.com/docs/content-configuration
